src/app/components/Paragraph/index.stories.tsx

```diff
@@ -1,90 +1,80 @@
 import React from 'react';
-import { withKnobs, select } from '@storybook/addon-knobs';
 
-import { ServiceContextProvider } from '../../contexts/ServiceContext';
-import { withServicesKnob } from '../../legacy/psammead/psammead-storybook-helpers/src';
-import ThemeProvider from '../ThemeProvider';
+import { FontVariant, GelFontSize } from '#app/models/types/theming';
 import Paragraph from '.';
-import md from './README.md';
+import readme from './README.md';
 import { StoryProps } from '../../models/types/storybook';
 
 interface Props extends StoryProps {
   text: string;
+  fontVariant: FontVariant | '--';
+  size: GelFontSize | '--';
 }
 
 const EMPTY_OPTION = '--';
```

Comment:
Can you use a const within a type?

```suggestion
const EMPTY_OPTION = '--';

interface Props extends StoryProps {
  text: string;
  fontVariant: FontVariant | EMPTY_OPTION;
  size: GelFontSize | EMPTY_OPTION;
}


```